dm/syncer/syncer.go

```diff
@@ -1321,6 +1323,27 @@ func (s *Syncer) syncDDL(queueBucket string, db *dbconn.DBConn, ddlJobChan chan
 		if !ok {
 			return
 		}
+
+		// set timezone
+		if ddlJob.timezone != "" {
+			s.timezone_last_time = ddlJob.timezone
+			setTimezoneSQL := fmt.Sprintf("SET SESSION TIME_ZONE = '%s'", ddlJob.timezone)
+			ddlJob.ddls = append([]string{setTimezoneSQL}, ddlJob.ddls...)
+			setTimezoneSQLDefault := fmt.Sprintf("SET SESSION TIME_ZONE = DEFAULT")
+			ddlJob.ddls = append(ddlJob.ddls, setTimezoneSQLDefault)
+		} else {
```

Comment:
> For two sequential DDL, if the latter one uses the same timezone as the former one, binlog will not write timezone again.

should check `s.timezone_last_time` in case we start task in the middle of 2 ddl, one has timezone, one not

Reply:
Yes. I have modified it as it will use s.timezone_last_time if current timezone is "".

Reply:
`s.timezone_last_time` might be empty in this case, should check it